pom.xml

```diff
@@ -68,7 +68,7 @@
 
   <properties>
     <rename.offset>org.apache.phoenix.thirdparty</rename.offset>
-    <guava.version>32.1.1-android</guava.version>
+    <guava.version>32.1.2-jre</guava.version>
```

Comment:
32.1.3-jre released just last week. let me know if should bump

Reply:
I see no reason not to use the latest.

Reply:
Updated!